tox.ini

```diff
@@ -11,7 +11,7 @@ deps =
     django20: Django>=2.0,<2.1
     django21: Django>=2.1,<2.2
     django22: Django>=2.2,<2.3
-    -rtest_requirements.txt
+    -rrequirements/tests.txt
```

Comment:
It looks like the non-Django base dependencies from `setup.py` aren't pinned anywhere; shouldn't they be, at least for tests?